components/brave_wallet_ui/panel/style.ts

```diff
@@ -29,6 +29,6 @@ export const SendWrapper = styled.div`
   overflow-y: scroll;
   overflow-x: hidden;
   position: relative;
-  padding: 0px 24px 12px;
+  padding: 0px 12px 12px;
```

Comment:
I think `Send` and `Swap` are also wrapped with this in the panel `Container`, how is this style change effecting them?

Reply:
@Douglashdaniel

**Before:**
<img width="326" alt="Screenshot 2022-04-13 at 19 58 21" src="https://user-images.githubusercontent.com/48117473/163233037-28c86b66-2d9e-4486-b35b-1d02f89bf247.png">

**After:**
<img width="322" alt="Screenshot 2022-04-13 at 20 02 57" src="https://user-images.githubusercontent.com/48117473/163233089-f097373a-4e0b-4e66-a2b6-46661c6298b2.png">

Reply:
Was talking about the `Send` and `Swap` panels since they are also wrapped in that `style`.